jenkins/cron-file.txt

```diff
@@ -0,0 +1,30 @@
+# This file can be used to apply the crontab for a given user
+# Edit this file to introduce tasks to be run by cron.
+#
+# Usage : crontab cron-file.txt
+# 
+# Each task to run has to be defined through a single line
+# indicating with different fields when the task will be run
+# and what command to run for the task
+#
+# To define the time you can provide concrete values for
+# minute (m), hour (h), day of month (dom), month (mon),
+# and day of week (dow) or use '*' in these fields (for 'any').#
+# Notice that tasks will be started based on the cron's system
+# daemon's notion of time and timezones.
+#
+# Output of the crontab jobs (including errors) is sent through
+# email to the user the crontab file belongs to (unless redirected).
+#
+# For example, you can run a backup of all your user accounts
+# at 5 a.m every week with:
+# 0 5 * * 1 tar -zcf /var/backups/home.tgz /home/
+#
+# For more information see the manual pages of crontab(5) and cron(8)
+# 
+# m h  dom mon dow   command
+
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
+
+00 05 * * * cd ~/openebs && ./tar_images.sh
```

Comment:
Just a suggestion, can we run this cron job once in 2 weeks rather than daily, we know that its not going to have changes frequently, lets not try to get images on daily basis. 

I am open for discussion on this, if its required.

Reply:
@yudaykiran , if there are no updates to the tarball (image) versions, the playbooks skips the steps that fetch/tar images (The output for the skipped steps shows the playbook run completing in 10s). So, it should still be ok to run it everyday v/s once in 2 weeks where the window to detect change maybe much larger.

Reply:
LGTM